xml/admin_about.xml

```diff
@@ -19,195 +19,304 @@
-      data or which OSD it needs to place the data.
-     </para>
-     <para>
-      CRUSH maintains a hierarchy of devices and the replica placement policy.
-      As new devices are added, data from existing nodes is moved to the new
-      device to improve distribution with regard to workload and resilience. As
-      a part of the replica placement policy, it can add weights to the devices
-      so some devices are more favored as opposed to others. This could be used
-      to give more weights to Solid State Devices (SSDs) and lower weights to
-      conventional rotational hard disks to get overall better performance.
-     </para>
-     <para>
-      CRUSH is designed to optimally distribute data to make use of available
-      devices efficiently. CRUSH supports different ways of data distribution
-      such as the following:
-     </para>
-     <itemizedlist mark="bullet" spacing="normal">
+   <variablelist>
+    <varlistentry>
+     <term>&ceph; Monitor</term>
+     <listitem>
+      <para>
+       Monitoring (MON) nodes maintain information about cluster health state, a
+       map of the cluster and placement rules. The placement rules are used to
+       improve the resilience of the cluster, for example by distributing object
+       replications to different server racks or data centers.
+      </para>
+      <para>
+       If conflicts occur, the monitor nodes in cluster decide by majority which
+       information is correct. To be able to form a majority, at least 3 monitor
+       nodes are needed. Monitor nodes also keep history of changes performed
+       to the cluster.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>&ceph; OSD</term>
+     <listitem>
+      <para>
+       A &ceph; OSD is a daemon handling one or more object-based storage devices.
+       Object-based storage devices can be physical disks/partitions or logical
+       volumes. The daemon additionally takes care of data replication and 
+       rebalancing in case of added or removed nodes. 
+      </para>
+      <para>
+       OSD daemons communicate with monitor nodes and provide them with the
+       state of the other OSD daemons.
+      </para>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+   <note>
+    <title>Each Node Type on a Separate Server</title>
+    <para>
+     We strongly recommend to install only one node type on a single server.
+    </para>
+   </note>
+   <para>
+    The software that provides all required object store functions is called 
+    <emphasis>RADOS</emphasis> (Reliable Autonomic Distributed Object Store).
+   </para>
+  </sect2>
+  <sect2 xml:id="storage.intro.access">
+   <title>Accessing Data</title>
+   <para>
+    &ceph; provides 4 methods of accessing data. The access methods
+    cover many different use cases. The underlying software providing
+    the functionality is RADOS. The 4 methods are:
```

Comment:
are you sure that cephfs is backed by RADOS? and how about iSCSI? it's missing here somehow...